README.md

```diff
@@ -37,6 +37,12 @@ features like flow control, data structures, and functions, with high-level
 constructs like objects, protocols, closures, and generics. Swift embraces
 modules, eliminating the need for headers and the code duplication they entail.
 
+Announced in 2014, the Swift programming language has quickly become one of the fastest growing languages in history.
+Swift makes it easy to write software that is incredibly fast and safe by design
+Swift is a powerful and intuitive programming language for iOS, iPadOS, macOS, tvOS, and watchOS.
+Writing Swift code is interactive and fun, the syntax is concise yet expressive, and Swift includes modern features developers love. 
+Swift code is safe by design, yet also produces software that runs lightning-fast.
```

Comment:
This reads like an ad, is it really needed?

Reply:
Hey @buttaface Thanks for reviewing I would like keep my point for your Query Asked 

No it dosen't its just like providing some feature of Swift that is needed incase to understand advantages of swift over other Languages